zephyr/zephyr.cmake

```diff
@@ -48,6 +48,16 @@ if(CONFIG_PICOLIBC_USE_MODULE)
     endif()
   endfunction()
 
+  function (picolibc_option_any_true name option_list)
```

Comment:
I'm not sure if this is meaningfully nicer than just doing an `if (SIZE_OPTS OR MIN_SIZE_OPTS) ... else ...` but this seemed in line with the more general `picolibc_option*` functions.

I'm a bit new to cmake yet so apologies in advance if there is a more "canonical" way of doing this and/or if a simple if/else would be sufficient--I'm happy to fix this as needed!

Reply:
whatever works; this seems fine to me. cmake is not something I know well.